tests/testthat/examples/depends-query/depends-query.R

```diff
@@ -1,6 +1,6 @@
-orderly2::orderly_parameters(a = NULL, b = NULL, c = NULL)
+pars <- orderly2::orderly_parameters(a = NULL, b = NULL, c = NULL)
 
-value <- c
```

Comment:
I am curious about how `pars$a`, and `pars$b` can be referenced as `this:a` and `this:b` on lines 6 and 7 - how does it know it's `pars` ? (Is this residue-magic?)

Reply:
this is magic that remains - the orderly report "knows" what its parameters are - the only change here is how user code finds that out